action.yaml

```diff
@@ -82,6 +82,10 @@ inputs:
   build-secrets:
     required: false
     description: "Mimics docker build --secret flag. See https://docs.docker.com/build/building/secrets/ for more information. Example input 'id=mysecret,src=secrets.txt'"
+  submodule:
```

Comment:
How about `checkout-submodules` so its more clearly about the checkout and that you could have multiple?

Reply:
updated to `checkout-submodule`, also ran through some manual tests to ensure it's functioning as per expectation